site/content/docs/5.2/forms/checks-radios.md

```diff
@@ -363,80 +363,74 @@ Drop borders using `.btn-no-outline`, too.
 Star rating system is built on top of radios. Simply add `.star-rating` to a `<fieldset>` element to use predefined glyphs and compose your star rating system with as much stars as needed.
 
 {{< example >}}
-<form>
```

Comment:
Consistency + no need of a form in here since there isn't any submit in here -> Ask to a11y expert.